### PR TITLE
Content and typo fixes

### PR DESCRIPTION
**Pull request by maximevaillancourt**
- Translated English elements to French
- Fixed typos
- Reworked sentence structure

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -630,16 +630,16 @@ <h2 class="large section-title">Notre plan de commandites</h2>
                             </div><!--pricing wrapper-->
                         </div><!--price col-->
                         <div class="col-md-4">
-                            <div class="pricing-wrapper wow animated fadeInRight" data-wow-delay=".9s">
+                            <div class="pricing-wrapper price-popular wow animated fadeInUp" data-wow-delay=".6s">
                                 <div class="pricing-wrapper-inner">
-                                    <h1>i5</h1>
-                                    <h2>1000 $</h2>
+                                    <h1>i7</h1>
```
Comment (DrPandemic) [inline index.html:635]: Thanks for the reordering. We should ask @Sytten to see if we could have another element for the i7. I don't like the fact that i5 has 5 elements and i7 has 4 and a `x` at the end. It's just about the presentation.

**Comment by DrPandemic** [comment]
LGTM :+1:

File: index.html (hunk 1 above)
Comment (maximevaillancourt) [inline index.html:635]: Agreed! Also, IMHO, the X implies that a feature is not included. You know, something like this.

2015-12-28-200724_664x239_scrot

How about we just remove the X (or replace it with something that doesn't imply a disadvantage)?

Comment (DrPandemic) [inline index.html:635]: Let's open an issue for this

**Pull request merged**